tests/test_replies/test_context.py

```diff
@@ -16,32 +23,43 @@ def test_reply_context_1(self):
         assert reply_context.post_html
         assert reply_context.post_text
 
+    @responses.activate
     def test_reply_context_2(self):
-        reply_context = context.get_reply_context(url="https://aaronparecki.com/2022/01/29/12/raspi-usb-webcam-hdmi")
+        url = "https://aaronparecki.com/2022/01/29/12/raspi-usb-webcam-hdmi"
+        author_url = "https://aaronparecki.com/"
+        with open("tests/fixtures/reply2.html") as f:
+            responses.add(responses.Response(responses.GET, url=url, body=f.read()))
+
+        with open("tests/fixtures/author2.html") as f:
+            responses.add(responses.Response(responses.GET, url=author_url, body=f.read()))
+
+        reply_context = context.get_reply_context(url=url)
 
         assert reply_context.post_url == "https://aaronparecki.com/2022/01/29/12/raspi-usb-webcam-hdmi"
         assert reply_context.webmention_endpoint == "https://webmention.io/aaronpk/webmention"
-        assert reply_context.authors[0].url == "https://aaronparecki.com/"
+        assert reply_context.authors[0].url == author_url
         assert reply_context.authors[0].name == "Aaron Parecki"
         assert reply_context.authors[0].photo == "https://aaronparecki.com/images/profile.jpg"
         assert reply_context.video == ""
         assert reply_context.photo == ""
         assert reply_context.post_html
         assert reply_context.post_text
 
+    @responses.activate
     def test_reply_context_3(self):
-        reply_context = context.get_reply_context(
-            url="https://www.theguardian.com/technology/2022/jan/31/beats-fit-pro-review-apple-workout-ready-airpods-pro-rivals-battery-price"  # noqa
+        url = (
+            "https://www.theguardian.com/technology/2022/jan/31/"
+            "beats-fit-pro-review-apple-workout-ready-airpods-pro-rivals-battery-price"
         )
+        with open("tests/fixtures/reply3.html") as f:
+            responses.add(responses.Response(responses.GET, url=url, body=f.read()))
+        reply_context = context.get_reply_context(url=url)
 
-        assert (
-            reply_context.post_url
-            == "https://www.theguardian.com/technology/2022/jan/31/beats-fit-pro-review-apple-workout-ready-airpods-pro-rivals-battery-price"  # noqa
-        )
+        assert reply_context.post_url == url
         assert reply_context.webmention_endpoint == ""
         assert reply_context.authors[0].url == "https://www.theguardian.com"
         assert reply_context.authors[0].name == ""
-        assert reply_context.authors[0].photo == "https://static.guim.co.uk/images/favicon-32x32.ico"
+        assert reply_context.authors[0].photo == ""
```

Comment:
It seems that their html must have changed as the favicon was no longer being set for the author url.